pyOCD/board/mbed_board.py

```diff
@@ -35,6 +35,7 @@
                 "9004": "lpc1768",
                 "1040": "lpc11u24",
                 "1050": "lpc800",
+                "1060": "lpc4088",
```

Comment:
What is the name of this board? The "EA LPC4088 QuickStart Board"?  I'm adding board name in #129